plugins/builds/index.js

```diff
@@ -27,30 +27,46 @@ const workflowParser = require('screwdriver-workflow-parser');
  * @param  {Boolean}  [config.start]        Whether to start the build or not
  * @return {Promise}
  */
```

Comment:
Need to add `pipelineFactory` to JSDoc

Reply:
We didn't need pipelineFactory, so I deleted.